utils/deployContract.js

```diff
@@ -1,56 +1,29 @@
 'use strict';
 
-const DeployContract = function(params) {
+const DeployContract = function (params) {
   const oThis = this;
```

Comment:
space is not needed after function. review and correct formatting all over the file.